build.gradle

```diff
@@ -63,8 +63,8 @@ android {
 
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_7
-        targetCompatibility JavaVersion.VERSION_1_7
+        sourceCompatibility JavaVersion.VERSION_1_5
```

Comment:
This won't be valid. We will move soon to Material Design. That requieres Java 7, according to Google documentation.